plugins/SLURM-Integration/install.sh

```diff
@@ -67,7 +67,7 @@ function update_slurm_conf()
 function set_python3_path()
 {   local full_python3_path=$1
     local file_name=$2
-    sed -i -e "s#python3_path#$full_python3_path#g" $SLURM_DIR/$file_name
+    sudo sed -i -e "s#python3_path#$full_python3_path#g" $SLURM_DIR/$file_name
```

Comment:
why no exit with error code after line 72? or it is in diff place?

Reply:
we don't want to exit the installation in case of failure, because we are notifying the user about the failure about the failure and we are asking him to do it manually. you can fine it the error message in line 72.

Reply:
we should exit on 1st error and not continue, so user will see error, fix and re-run
also - why sudo is inside script and not outside
also need to check that $file_name is writeable and fail if not
check if token #python3_path# exists in file and do sed only when it does and provide clear error what is wrong

Reply:
working on the fixes, regarding the sudo I tried to remove it from the script and run sudo ./install.sh but we faced many issues with permission, so decided to keep the sudo inside the script.

Reply:
what issues? using sudo inside the script is not a good approach, as not every slurm install allows/has sudo and admin will need to change script and remove sudo as cmd